zhaquirks/sonoff/s60zbtpf.py

```diff
@@ -0,0 +1,150 @@
+"""Sonoff S60ZBTPF - Zigbee Smart Plug."""
+
+from typing import Final
+
+from zigpy.quirks import CustomCluster
+from zigpy.quirks.v2 import QuirkBuilder
+from zigpy.quirks.v2.homeassistant import (
+    UnitOfElectricCurrent,
+    UnitOfElectricPotential,
+    UnitOfPower,
+)
+from zigpy.quirks.v2.homeassistant.number import NumberDeviceClass
+import zigpy.types as t
+from zigpy.zcl import ClusterType, foundation
+import zigpy.zcl.foundation as zcl_f
+from zigpy.zcl.foundation import BaseAttributeDefs, ZCLAttributeDef
+
+
+class SonoffCluster(CustomCluster):
+    """Custom Sonoff cluster."""
+
+    cluster_id = 0xFC11
+    manufacturer_id_override: t.uint16_t = foundation.ZCLHeader.NO_MANUFACTURER_ID
+
+    class AttributeDefs(BaseAttributeDefs):
+        """Attribute definitions."""
+
+        outlet_control_protect_setting = ZCLAttributeDef(
+            name="outlet_control_protect_setting",
```

Comment:
The `name` doesn't need to be provided here, since we're already using the new-style `AttributeDefs`.
We'll just use the actual attribute ("value") name.